### DISCUSSION TITLE
没有分类(categories) icon

### DISCUSSION THREAD

**Discussion by OnqiauJliu** [hugo-next/hugo-theme-next#105] [Q&A] [answered]
请按照此新功能模版提供相关信息，如详细的功能描述或同类效果屏幕截图，这将有助于我们进行调查。
(Please follow this new feature template to provide relevant information, such as detail description for feature or some screenshots, which will help us investigate.)

## 新功能描述 (New feature description)
目前在菜单栏中没有 categories 的icon  希望能添加一个
![image](https://github.com/hugo-next/hugo-theme-next/assets/60482496/6dbf2071-4345-4bff-b7e7-e91f4fbdea6e)
![image](https://github.com/hugo-next/hugo-theme-next/assets/60482496/412e28c9-6529-4567-b67b-44baee8088c4)

**Comment by elkan1788** [comment, marked as answer]
@OnqiauJliu 可以直接在 [FontAwesome](https://fontawesome.com/) 上面找一个你喜欢的 **icon** 图标，注意是要使用免费版本的。

**Reply by OnqiauJliu** [reply]
能添加一个默认的吗？  自己添加的话 在配置文件上看起来就没那么统一

**Reply by elkan1788** [reply]
后续可以考虑提示个示例参考吧
